paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -159,6 +159,11 @@ def paasta_start_or_stop(args, desired_state):
     else:
         clusters = list_clusters(service)
```

Comment:
Haven't changed this here but this line actually makes me a little nervous. Defaulting to all the instances for a service seems safe enough. But defaulting to all clusters makes me worried I might start/stop things in prod just by forgetting to specify a cluster? Anyone else think we should change this?